user/service/client/client.go

```diff
@@ -202,6 +203,14 @@ func (c *Client) Delete(ctx context.Context, id string, deleet *user.Delete, con
 			logger.WithError(err).Error("Unable to destroy profile")
 		}
 	}
+	//Sending delete user event to kafka
+	var role []string
+	if result != nil && result.Roles != nil {
+		role = *result.Roles
+		logger.Infof("PResult: %v", *result.Roles)
+	}
+	logger.Infof("PResult: %v", *result)
+	c.CloudEventsClient().KafkaMessage("delete-user", id, *result.Username, role)
```

Comment:
Consider extracting "delete-user" event name to a new constant.

Reply:
addressed in changes recent commit